core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -2183,6 +2183,13 @@ object SparkContext extends Logging {
   }
 
   /**
+   * Return SparkContext
+   */
+  def get(): SparkContext = {
```

Comment:
@holdenk, figured this may be a good utility to have in SparkContext alongside getOrCreate. Unless you think otherwise?